pkgs/by-name/os/ossia-score/package.nix

```diff
@@ -130,7 +130,7 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://ossia.io/score/about.html";
     description = "A sequencer for audio-visual artists, designed to enable the creation of interactive shows, museum installations, intermedia digital artworks, interactive music and more";
     # TODO: this should work for darwin too
-    platforms = platforms.linux;
+    platforms = "x86_64-linux";
```

Comment:
@minijackson it seems that this change broke the build ?

Reply:
Oh right sorry, it's supposed to be a list:

```nix
    platforms = [ "x86_64-linux" ];
```